Dockerfile.deploy.rhel

```diff
@@ -0,0 +1,21 @@
+FROM prod.registry.devshift.net/osio-prod/base/pcp:latest
+LABEL maintainer "Devtools <devtools@redhat.com>"
+LABEL author "Devtools <devtools@redhat.com>"
+ENV LANG=en_US.utf8
+ENV INSTALL_PREFIX=/usr/local/fabric8-osoproxy
+
+# Create a non-root user and a group with the same name: "fabric8"
+ENV F8_USER_NAME=fabric8
+RUN useradd --no-create-home -s /bin/bash ${F8_USER_NAME}
+
+COPY traefik ${INSTALL_PREFIX}/bin/fabric8-osoproxy
```

Comment:
@jmelis this should be `dist/traefik` (/me referring the CentOS based Dockerfile)

https://github.com/fabric8-services/fabric8-oso-proxy/blob/c3ca6b9de1fa17f589d397da1c325fb704891121/Dockerfile.deploy#L10-L11

Reply:
very good catch! thanks!